be/src/storage/rowset/context_reader.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "column/schema.h"
+#include "common/status.h"
+#include "storage/rowset/column_iterator.h"
+#include "storage/rowset/common.h"
+
+namespace starrocks::vectorized {
+class Chunk;
+class SparseRange;
+
+class ColumnCollector {
+public:
+    ColumnCollector(std::vector<ColumnIterator*> column_iterators) : _column_iterators(std::move(column_iterators)) {}
+    Status seek_columns(ordinal_t pos);
+    Status read_columns(Chunk* chunk, const SparseRange& range);
+
+private:
+    std::vector<ColumnIterator*> _column_iterators;
+};
+
+// some buffer
+class ScanContext {
+public:
+private:
+    // schema
+    // pool
+};
+
+class ReadTransfer {
```

Comment:
Could you give an explain about this interface.

Reply:
sorry this file is not relation with this PR